src/transaction.local.net.spec.ts

```diff
@@ -40,16 +39,16 @@ describe("test transaction", function () {
         let transactionOne = new Transaction({
             sender: alice.address,
             receiver: bob.address,
-            value: TokenTransfer.egldFromAmount(42),
-            gasLimit: network.MinGasLimit,
+            value: TokenTransfer.egldFromAmount(42).amount,
```

Comment:
let's not use this method

Reply:
I'll update in a different pr if that's ok